src/features/user/UserProfile/screens/PrivateIndividualProfile.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useRef } from 'react';
+import LandingSection from '../../../common/Layout/LandingSection';
+import styles from '../styles/UserProfile.module.scss';
+import Settings from '../../../../../public/assets/images/icons/userProfileIcons/Settings';
+import ScrollDown from '../../../../../public/assets/images/icons/userProfileIcons/ScrollDown';
+import MyForestContainer from '../components/MyForestContainer';
+import UserInfo from '../components/UserInfo';
+import Snackbar from '@material-ui/core/Snackbar';
+import MuiAlert, { AlertProps } from '@material-ui/lab/Alert';
+import SettingsModal from '../components/SettingsModal';
+
+export default function PrivateIndividualProfile({ privateUserprofile }: any) {
+
+  const [textCopiedsnackbarOpen, setTextCopiedSnackbarOpen] = React.useState(
+    false
+  );
+
+  const handleTextCopiedSnackbarOpen = () => {
+    setTextCopiedSnackbarOpen(true);
+  };
+  const handleTextCopiedSnackbarClose = (
+    event?: React.SyntheticEvent,
+    reason?: string
+  ) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+    setTextCopiedSnackbarOpen(false);
+  };
+
+  const scrollRef = useRef(null);
+
+  // settings modal
+  const [settingsModalOpen, setSettingsModalOpen] = React.useState(false);
+  const handleSettingsModalClose = () => {
+    setSettingsModalOpen(false);
+  };
+  const handleSettingsModalOpen = () => {
+    setSettingsModalOpen(true);
+  };
+
+  // editProfile modal  (from settings modal)
+  const [editProfileModalOpen, setEditProfileModalOpen] = React.useState(false);
+  const handleEditProfileModalClose = () => {
+    setEditProfileModalOpen(false);
+  };
+  const handleEditProfileModalOpen = () => {
+    setEditProfileModalOpen(true);
+  };
+
+  return (
+    <React.Fragment>
+      <main>
+        {/* will render only if it is ME page */}
+(
+          <>
+            <div
+              className={styles.settingsIcon}
+              onClick={handleSettingsModalOpen}
+            >
+              <Settings color="white" />
+            </div>
+            <SettingsModal
+              userprofile={privateUserprofile}
+              settingsModalOpen={settingsModalOpen}
+              handleSettingsModalClose={handleSettingsModalClose}
+              editProfileModalOpen={editProfileModalOpen}
+              handleEditProfileModalClose={handleEditProfileModalClose}
+              handleEditProfileModalOpen={handleEditProfileModalOpen}
+            />
+          </>
+        )
+
+        {/* userinfo section */}
+        <LandingSection
+          imageSrc={
+            process.env.CDN_URL
+              ? `${process.env.CDN_URL}/media/images/app/bg_layer.jpg`
+              : `https://cdn.plant-for-the-planet.org/media/images/app/bg_layer.jpg`
```

Comment:
I think having the right env values is a must have for our software. We do not have to code any urls as fallbacks into the source which might be pretty soon be outdated.